lpc17prl.lua

```diff
@@ -172,6 +174,32 @@ if opts.mode == 'read' then
 end
 
 
+if opts.mode == 'calculate' then
+  if not opts.fname then D.abort(2, "Binary file not provided") end
+
+  local f, err = io.open(opts.fname, "r+b")
+  if not f then D.abort(2, "Cannot open input file: "..opts.fname..": "..err) end
+
+  local sum = 0
+
+  for byteno=1,7,1 do
+    local bytes = f:read(4)
+    local int32 = B.dec32LE(bytes)
+    sum = sum + int32
+    D.yellow''(byteno,int32,B.hex(int32))
```

Comment:
Although it's convenient for development purposes - I think user is not interested in those values